src/box/index_def.h

```diff
@@ -151,8 +151,16 @@ struct index_opts {
 	 * previous one.
 	 */
 	double run_size_ratio;
-	/* Bloom filter false positive rate. */
+	/* Bloom filter false positive rate.
+	 * Used only as default parameter when rebuilding index
+	 * See vy_run_rebuild_index()
+	 */
 	double bloom_fpr;
+    /* Coefficient of worst-case read scenario:
+     * the lower parameter, the higher read performance
+     * with more memory consumption.
+     */
```

Comment:
Please elaborate on the comment — it should rather be 'mean amount of disk lookups'.

Reply:
Should I still change the comment as we discussed that `lookup_cost_coeff` is not a mean amount of lookups?

Reply:
Yes.

Reply:
Done.